keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -2518,6 +2648,20 @@ For Beta and GA, add links to added tests together with links to k8s-triage for
 https://storage.googleapis.com/k8s-triage/index.html
 -->
 
+The scheduler plugin and resource claim controller are covered by the workloads
```

Comment:
Please update the coverage percentages for all involved packages (new and old)

IIRC, the coverage was pretty low for alpha.

Let's make it a goal for beta graduation to reach 80%

Reply:
Several PRs have already gone in to increase coverage. I'll create a KEP update with the latest numbers that we can review before merging the beta promotion PR.

Reply:
https://github.com/kubernetes/kubernetes/issues/127949